libs/perun/components/src/lib/members-list/members-list.component.ts

```diff
@@ -63,9 +69,10 @@ export class MembersListComponent implements OnChanges, AfterViewInit {
   @Input() tableId: string;
   @Input() disableRouting = false;
   @Input() filter = '';
+  @Input() consents: Consent[] = [];
```

Comment:
After the changes we do not need the consents anymore. 
Other than that the filtering and exporting is working properly so well done.